Makefile

```diff
@@ -115,6 +115,10 @@ build-platform-assets:
 	@echo "--> Building platform assets"
 	@echo "from sentry.utils.integrationdocs import sync_docs; sync_docs(quiet=True)" | sentry exec
 
+build-chartcuterie-config:
```

Comment:
It does build
<img width="863" alt="image" src="https://user-images.githubusercontent.com/44410/112210494-20e21a00-8bf1-11eb-8e32-a5b037d1d022.png">

and it produces a file:
```
❯ ls -l config/chartcuterie/config.js
-rw-r--r--  1 armenzg  staff  3293520 23 Mar 16:00 config/chartcuterie/config.js
```